### ISSUE TITLE
Refresh Features

### ISSUE THREAD

**Issue by jeffreykwon44** [val92130/Supreme-Auto-Checkout#31]
-It would be nice to be able to change the refresh interval just in case you would want to. 
-Proxy support for multiple tasks
-An accompanying mobile app or mobile site to solve captchas and see order from somewhere else if you don't have access to computer during drop but you have access to phone
-A mobile full fledged bot

P.S Great bot. You are an amazing human for making this public

**Comment by val92130** [comment]
Thanks for the great ideas!

Proxy support and a mobile app is scheduled but it's taking me a lot of time so I'm looking for contributors to help me out on that.

Also more stores are coming, the bot will soon be renamed "OpenAIO" and I will progressively add more websites, Palace should be the first :)

The Autocop is based on your computer time, it can cop multiple items but I recommend to run it manually using the "run now" button for each products on your Autocop products list if you want to cop multiple items!

**Comment by jeffreykwon44** [comment]
I don't know much about programming, but don't most shopify sites like Palace and Undefeated and Bape have the same code or similar code. It might be easier for you to find the shopify code but I don't know much about code or bots. 
Do you think you would ever make this bot an application like an .exe or .jar or would you keep it a chrome extension?

**Comment by val92130** [comment]
Yes you’re right! I think it’s much easier to get products on Shopify websites. 

I don’t think this bot will ever be released as and executable sorry :/ or maybe as a stand-alone chrome app but I haven’t looked at that for the moment!

**Comment by jeffreykwon44** [comment]
you should add an option in the color for "any"

**Comment by val92130** [comment]
If you don't add a color it chooses any color for the autocop :)

**Comment by jeffreykwon44** [comment]
Do you think you can add a yeezy supply or adidas option by sunday? The calabasas pants are dropping

**Comment by val92130** [comment]
No sorry this would take a lot of time :/

Right now I'm working fully on the Supreme part to make sure everything works fine for the Stone Island drop :)

**Comment by manbearcoon** [comment]
+1 on proxy support. Really looking forward to testing that.

**Comment by jeffreykwon44** [comment]
Nice I'll check it out

**Comment by jeffreykwon44** [comment]
What is "Retry Count if not found"?
If I really want to cop the headband what should i put there?
